libsplinter/src/biome/rest_api/actix/user.rs

```diff
@@ -153,6 +153,7 @@ fn add_modify_user_method(
     Box::new(move |request, payload| {
         let credentials_store = credentials_store.clone();
         let key_store = key_store.clone();
+        let rest_config = rest_config.clone();
```

Comment:
You could probaly just clone the encryption cost, instead of the whole configuration.  Additionally, you could make the encryption struct `Copy` and avoid the use of clone.